### PR TITLE
fix(owners): Suggested assignees may now be clicked again

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This regressed in 738c5262dd

Fixes [PROD-100](https://getsentry.atlassian.net/browse/PROD-100)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 1]
```diff
@@ -181,12 +181,13 @@ const SuggestedOwners = createReactClass({
                   commits={owner.commits}
                   containerClassName="avatar-grid-item"
                 >
-                  <ActorAvatar
-                    style={{cursor: 'pointer'}}
-                    hasTooltip={false}
-                    actor={owner.actor}
-                    onClick={() => this.assignToActor(owner)}
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: Was this not working?

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 2]
```diff
@@ -181,12 +181,13 @@ const SuggestedOwners = createReactClass({
...
-                  />
+                  <span onClick={() => this.assign(owner.actor)}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: Is this equivalent? Previously it was calling `assignToActor(owner)`

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: Not at all

[the remark this answers is not in the document]
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: https://github.com/getsentry/sentry/blob/a79ee79538625487fa27a4002027db111c8e2149/src/sentry/static/sentry/app/components/avatar/baseAvatar.jsx#L146-L159

Actually not too bad I think. I'll revisit

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: Yeah, it was wrong. There was `assignToActor` and `assignToUser`, the later never being used. and `assingToActor` only being used for team assignment

**Review by lynnagara — APPROVED** [review]

**Pull request merged**